public/css/login-registration.css

```diff
@@ -17,7 +19,7 @@
 
 /* GOOGLE LOGIN */
 hr {
-	border-top-color: #2b2b2b;
+	border-top-color: var(--light-green);
```

Comment:
`var(--light-green)` is actually no longer used in the design, so we shouldn't use it. I pushed the commit to `style.css` before Courtney made the change, but I'll remove it soon. Just switch to `var(--green)`